apis/project.cattle.io/v3/schema/schema.go

```diff
@@ -529,13 +529,6 @@ func podTypes(schemas *types.Schemas) *types.Schemas {
 			m.Move{From: "readOnlyRootFilesystem", To: "readOnly"},
 			m.Move{From: "runAsUser", To: "uid"},
 		).
-		AddMapperForType(&Version, v1.Container{},
-			m.Move{From: "command", To: "entrypoint"},
-			m.Move{From: "args", To: "command"},
-			mapper.EnvironmentMapper{},
-			&m.Embed{Field: "securityContext"},
-			&m.Embed{Field: "lifecycle"},
-		).
```

Comment:
We can't drop this whole block, just the mapper that tied to the envvars

Reply:
`mapper.EnvironmentMapper{}`

Reply:
I was about to write a comment asking about this. I'll update it, thanks!